org.gnome.design.Emblem.json

```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "org.gnome.design.Emblem",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41rc",
```

Comment:
shall we wait for 41 to be out?

Reply:
If it is a blocker it could be set to 40 in the meanwhile.

Reply:
I mean I can use 40 in the meantime. Libadwaita actually requires stuff from 41, but for all I have seen it works. I am waiting on the CI to resolve this and can be updated in a week.

Reply:
Tested and works.